sdk/servicebus/Azure.Messaging.ServiceBus/tests/ServiceBusLiveTestBase.cs

```diff
@@ -1,14 +1,12 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
-using System.Collections.Generic;
-using System.Text;
-using Azure.Core.Testing;
+using NUnit.Framework;
 
 namespace Azure.Messaging.ServiceBus.Tests
 {
-    [LiveOnly]
```

Comment:
This has ties to the HTTP test recording framework beyond applying the category; since there's no equivalent recording for AMQP and Service Bus is explicitly opting-out of the recording framework, I felt it was better to convert this over to a pure set of category metadata.